app/controllers/AccountManagement.scala

```diff
@@ -189,9 +191,10 @@ object ManageWeekly extends ContextLogging {
   }
 
   def apply(
-    billingSchedule: Option[BillingSchedule],
-    weeklySubscription: Subscription[WeeklyPlan],
-    promoCode: Option[PromoCode]
+             billingSchedule: Option[BillingSchedule],
```

Comment:
better to keep the standard indent where possible (to keep intellij happy)

Reply:
This is my standard intellij indentation 😦
Maybe we should try and get our .editorconf files correct or use something like scalafmt?